prow/plugins/trigger/trigger.go

```diff
@@ -99,7 +99,8 @@ func getClient(pc plugins.PluginClient) client {
 
 func handlePullRequest(pc plugins.PluginClient, pr github.PullRequestEvent) error {
 	trustedOrg := trustedOrgForRepo(pc.PluginConfig, pr.Repo.Owner.Login, pr.Repo.Name)
-	return handlePR(getClient(pc), trustedOrg, pr)
+	joinOrgURL := pc.PluginConfig.TriggerFor(pr.Repo.Owner.Login, pr.Repo.Name).JoinOrgURL
```

Comment:
`TriggerFor` can return nil. This will segfault for orgs that don't explicitly specify a trustedOrg.

Reply:
Opened https://github.com/kubernetes/test-infra/pull/5978.